packages/next/build/entries.ts

```diff
@@ -177,10 +176,10 @@ export function createEntrypoints(
         name: '[name].js',
         value: `next-middleware-ssr-loader?${stringify({
           page,
+          stringifiedConfig: JSON.stringify(config),
           absolute500Path: pages['/500'] || '',
           absolutePagePath,
           isServerComponent: isFlight,
-          serverComponents: hasServerComponents,
```

Comment:
do you also need `dev: false` or leave it default `false` since it's going to be stringified

Reply:
It will be serialized as `undefined` so it's fine.